lib/sqlalchemy/engine/row.py

```diff
@@ -83,7 +88,7 @@ def __setattr__(self, name: str, value: Any) -> NoReturn:
     def __delattr__(self, name: str) -> NoReturn:
         raise AttributeError("can't delete attribute")
 
-    def _tuple(self) -> _TP:
+    def _tuple(self) -> Tuple[Unpack[_Ts]]:
```

Comment:
**Federico Caselli** (CaselIT) wrote:

I guess we can also deprecate this

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986

Reply:
Thanks, done